packages/iris-grid/src/formatters/NumberColumnFormatter.test.ts

```diff
@@ -44,22 +45,22 @@ numberColumnFormatters.forEach(({ name, formatter: NumberColumnFormatter }) => {
     const formatter = new NumberColumnFormatter();
     it('does not throw error for invalid formatString in format argument', () => {
       expect(() => {
-        formatter.format(INVALID_FORMAT, 0);
+        formatter.format(0, INVALID_FORMAT);
       }).not.toThrow();
     });
 
     it('does not throw error for invalid value argument', () => {
       expect(() => {
-        formatter.format(null, null);
+        formatter.format(null);
       }).not.toThrow();
     });
 
     it('returns empty string for invalid formatString in format argument', () => {
-      expect(formatter.format(INVALID_FORMAT, 0)).toBe('');
+      expect(formatter.format(0, INVALID_FORMAT)).toBe('');
     });
 
     it('returns empty string for null value ', () => {
-      expect(formatter.format(null, null)).toBe('');
+      expect(formatter.format(null)).toBe('');
```

Comment:
These are kind of weird tests. Could combine them, since they obviously didn't throw if they return a value?

Reply:
I'll just remove the 2 not throw tests